Completion/Unix/Command/_stow

```diff
@@ -0,0 +1,89 @@
+#compdef stow chkstow
+
+#
+# A zsh completion script for GNU stow (https://www.gnu.org/software/stow/)
+#
+
+__stow_complete_packages() {
+  local stow_dir=${(Q)1}
+  local -a stow_pkg_list=( $stow_dir/*(-/N:t) )
+
+  if [[ ${#stow_pkg_list} -gt 0 ]]; then
+    _values -C "packages from $stow_dir" ${stow_pkg_list[@]}
+  else
+    _message "no packages found in $stow_dir"
+  fi
+}
+
+case $service in
+  stow)
+    local context state line curcontext="$curcontext" ret=1
+    typeset -A opt_args
+    # Others local variables
+    local stow_dir arguments
+
+    arguments=(
+      '(- *)'{--help,-h}'[show help]'
+      '(- *)'{--version,-V}'[show version number]'
+      '(-d --dir)'{-d+,--dir=}'[set the stow dir (default is current dir)]:stow dir:_files -/'
```

Comment:
This doesn't seem to be in the completion style guide, but either Oliver or Daniel once suggested to me to put defaults in square brackets at the end of the match description, as in `stow directory [$PWD]`

Reply:
I don't see how to do that... could you give me an example please ?

Reply:
Change `:stow dir:` to `:stow dir [$PWD]:`.

One of us will add this guidance to Etc/completion-style-guide (or you can do that, of course).

Reply:
Done by @okdana (thanks!) in [workers/44969](http://www.zsh.org/cgi-bin/mla/redirect?WORKERNUMBER=44969).

Reply:
done in a3e659bdc